packages/ui-a11y-utils/src/FocusRegionOptions.ts

```diff
@@ -86,4 +86,8 @@ export type FocusRegionOptions = {
    * Whether or not the element is a Tooltip
    */
   isTooltip?: boolean
+  /**
+   * The ID of the `FocusRegion` this belongs to. Used only for debugging.
+   */
+  regionId?: string
```

Comment:
This just helps debug similar issues since it lets you see which focus region a component belongs to